.yamllint

```diff
@@ -0,0 +1,9 @@
+---
+extends: default
+
+rules:
```

Comment:
I would recommend adding additional constraints that will enforce white spaces to be consistent. Example: "spaces: 2" https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation

Ideally, this linting will both catch syntax errors and minimize diffs from PRs.